templates/STM32/Peripheral/GPIO2/TH.hs

```diff
@@ -26,16 +26,16 @@ mkGPIOPin portName pin n =
   ]
   where fs = [ varE portName
              , litE (integerL n)
-             , mkVar "gpio_mode_" n
-             , mkVar "gpio_otype_" n
-             , mkVar "gpio_ospeed_" n
-             , mkVar "gpio_pupd_" n
-             , mkVar "gpio_idr_" n
-             , mkVar "gpio_bs_" n
-             , mkVar "gpio_br_" n
+             , mkVar "gpio_moder_moder" n
+             , mkVar "gpio_otyper_ot" n
+             , mkVar "gpio_ospeedr_ospeedr" n
+             , mkVar "gpio_pupdr_pupdr" n
+             , mkVar "gpio_idr_idr" n
+             , mkVar "gpio_bsrr_bs" n
+             , mkVar "gpio_bsrr_br" n
```

Comment:
Dunno if this is right since both _bs and _br are actually in a single register in `GPIO2/Regs.hs`.